demo/src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
Just FYI if you move files with git, it would preserve the history and would mark file as moved rather than displaying it  as two different files.